### ISSUE TITLE
Parse function name

### ISSUE THREAD

**Issue by akoenig** [reduct/assembler#29]
The current implementation is not compatible with IE due to the lack oft support for `Function.name`.

I suggest to parse the actual name out of `Function.prototype.toString()` then.

**Comment by akoenig** [comment]
Implemented in ab69903da8cf965afff44d1c35630ecab8a8b115

It would be cooler to have this helper method in an own module (`Utilities`) with own specs and so on. To do this, we have to find a way in the `build-tools` to support writing the specs in `ES6` in order to import just the respective module (e.g. `Utilities`). We have to perform several steps to test an internal module atm: Bundle it and import this thingy in an ES5-written spec to perform the tests within the `build-tools` environment.

@Inkdpixels Can you elaborate on this again why it is needed to write the specs in ES5? Was it because of the `babel/register-hook` difficulty?

**Comment by Inkdpixels** [comment]
Ayo! 

Yeah makes sense @ moving this into a utility package.

Regarding the importing/bundling of dependencies - I've got this already on my list, as stated in https://github.com/reduct/build-tools/issues/30. I've got some plans/thoughts - But as of now I'm unsure about how to proceed here, we should discuss this together tomorrow/on tuesday.

And regarding the ES5 specs, thats definitely the issue with the babel/register hook - I think for writing ES6 specs, we need to wait for iojs/Node to catch up, as there are too many negative points in all solutions for the babel/register hook which could(!) work.
